mae_utils.py

```diff
@@ -68,3 +68,23 @@ def unflatten_tokens(
     img = x.reshape(shape=(x.shape[0], -1, h * patch_size, h * patch_size))
 
     return img
+
+def apply_norm_pix(flattened_imgs: torch.Tensor, eps: float = 1.0e-6) -> torch.Tensor:
```

Comment:
We can remove this as we do always set this hyperparameter to False!

Reply:
done